### ISSUE TITLE
Bug: deprecation warning on HA Core 2024.10.0b0

### ISSUE THREAD

**Issue by Anto79-ops** [hass-agent/HASS.Agent-Integration#11]
hello!

Running HASS.agent 2.1.0-beta1 and when updating HA Core to 2024.10.0b0 I noticed this in the logs:

```
2024-09-25 13:20:58.762 WARNING (ImportExecutor_0) [homeassistant.components.media_player.const] MEDIA_TYPE_MUSIC was used from hass_agent, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaType.MUSIC instead, please report it to the author of the 'hass_agent' custom integration
```

thanks all

**Comment by velodromen** [comment]
Same issue here

**Comment by Anto79-ops** [comment]
Thanks, confirmed fix in 2.1.1 beta 1
